trainee_template/react_series.amlg

```diff
@@ -491,16 +491,22 @@
 					(if (size (last series_data))
 						(let
 							(assoc
-								no_time_derived_action_features
-									(filter (lambda (!= tsTimeFeature (target_value))) derived_action_features)
+								initial_progress_context_features
+									(append
+										;use all value, delta, and rate features to determine initial series progress
+										;exclude time feature (and delta)
+										(filter (lambda (!= tsTimeFeature (target_value))) derived_action_features)
+										(filter (lambda (!= (concat "." tsTimeFeature "_delta_1") (target_value))) (get tsModelFeaturesMap "delta_features"))
```

Comment:
declare that concat'd time delta feature into a variable outside of this 'if' statement so this filter is comparing against a static value, thanks!